openedx/core/djangoapps/discussions/docs/decisions/0004-in-context-discussions-linking.rst

```diff
@@ -0,0 +1,131 @@
+Linking In-Context Discussions to Units
+=======================================
+
+
+Status
+------
+
+Proposal
+
+
+Context
+-------
+
+As part of the BD-38 initiative (Blended Development, Project 38), we are
+investigating a new way to set up discussions for a course that tries to
+simplify the way discussions are configured and ties them more closely to the
+course structure.
+
+The first simplification that the new system makes is to remove the
+discussions XBlock, and instead supports simply marking individual units
+as discussable, in which case a discussion UI will show up in-context.
+
+Currently this will only be supported for the edX Forums internal
+discussion provider, but it could be extended in the future to support
+other providers.
+
+In addition to providing a way to mark Units as discussable, there will
+also be a way to simply enable discussion for all units in the course.
+One further control provided here will be to toggle discussions on or
+off for graded discussions.
```

Comment:
So is it the case that the course-wide policy sets the default behavior, and individual units override that behavior? 

Say my course-wide setting of `"discussions_enable_graded_units": false,`, but I have a particular graded unit where I set the `discussable=true`. Does that mean that unit is discussable?

I think it's perfectly reasonable for that to be the case–I just want to make sure I understand correctly.

Reply:
Ack, nvm, it looks like it doesn't work this way. (looking at later text)